<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
      color: rgb(74, 74, 111);
	  background-color: rgb(168, 218, 220);
    
}
QPushButton {
 font: bold;
 font-size:12px;
 line-height:1.2;
  color: #f8f9fa;
  justify-content: center;
  align-items: center;
  padding: 0 20px;
  width: 100%;
  height: 50px;
  background-color: rgb(80, 80, 150);
  border-radius: 10px;

	
}

QPushButton:hover {
  background-color: rgb(74, 74, 125);
  box-shadow: 0 10px 30px 0px rgba(51, 51, 51, 0.5);
 
}

QPushButton:pressed {
   background-color: rgb(74, 74, 111);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>310</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-image: url();
    background-repeat: no-repeat;
    background-position: left;
    color: rgb(74, 74, 111);
	font-family:'Poppins-Medium', sans-serif;
    font-size: 18px;
    padding: 4 4 4 35; /* left padding (last number) must be more than the icon's width */
	border-radius:10px;
	border-bottom:2px solid #d9d9d9;
	background-color: rgb(241, 250, 238);
}

QLineEdit:hover{
    border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
    border: 2px solid rgb(85, 170, 255);
    background-color: rgb(241, 250, 238);
}</string>
    </property>
    <property name="placeholderText">
     <string>Paswward</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-image: url();
    background-repeat: no-repeat;
    background-position: left;
    color: rgb(74, 74, 111);
	font-family:'Poppins-Medium', sans-serif;
    font-size: 18px;
    padding: 4 4 4 35; /* left padding (last number) must be more than the icon's width */
	border-radius:10px;
	border-bottom:2px solid #d9d9d9;
	background-color: rgb(241, 250, 238);
}

QLineEdit:hover{
    border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
    border: 2px solid rgb(85, 170, 255);
    background-color: rgb(241, 250, 238);
}</string>
    </property>
    <property name="placeholderText">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>370</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-image: url();
    background-repeat: no-repeat;
    background-position: left;
    color: rgb(74, 74, 111);
	font-family:'Poppins-Medium', sans-serif;
    font-size: 18px;
    padding: 4 4 4 35; /* left padding (last number) must be more than the icon's width */
	border-radius:10px;
	border-bottom:2px solid #d9d9d9;
	background-color: rgb(241, 250, 238);
}

QLineEdit:hover{
    border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
    border: 2px solid rgb(85, 170, 255);
    background-color: rgb(241, 250, 238);
}</string>
    </property>
    <property name="placeholderText">
     <string>Paswward</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>200</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-image: url();
    background-repeat: no-repeat;
    background-position: left;
    color: rgb(74, 74, 111);
	font-family:'Poppins-Medium', sans-serif;
     font-size: 18px;
    padding: 4 4 4 35; /* left padding (last number) must be more than the icon's width */
	border-radius:10px;
	border-bottom:2px solid #d9d9d9;
	background-color: rgb(241, 250, 238);
}

QLineEdit:hover{
    border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
    border: 2px solid rgb(85, 170, 255);
    background-color: rgb(241, 250, 238);
}</string>
    </property>
    <property name="placeholderText">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>411</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;First Name :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(img/logo_mood.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>260</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-image: url();
    background-repeat: no-repeat;
    background-position: left;
    color: rgb(74, 74, 111);
	font-family:'Poppins-Medium', sans-serif;
    font-size: 18px;
    padding: 4 4 4 35; /* left padding (last number) must be more than the icon's width */
	border-radius:10px;
	border-bottom:2px solid #d9d9d9;
	background-color: rgb(241, 250, 238);
}

QLineEdit:hover{
    border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
    border: 2px solid rgb(85, 170, 255);
    background-color: rgb(241, 250, 238);
}</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins-Medium,sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-image: url();
    background-repeat: no-repeat;
    background-position: left;
    color: rgb(74, 74, 111);
	font-family:'Poppins-Medium', sans-serif;
    font-size: 18px;
    padding: 4 4 4 35; /* left padding (last number) must be more than the icon's width */
	border-radius:10px;
	border-bottom:2px solid #d9d9d9;
	background-color: rgb(241, 250, 238);
}

QLineEdit:hover{
    border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
    border: 2px solid rgb(85, 170, 255);
    background-color: rgb(241, 250, 238);
}</string>
    </property>
    <property name="placeholderText">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"> font-size:25px;
  color: rgb(74, 74, 111);
  justify-content: center;
  align-items: center;
  padding: 0 20px;
  width: 100%;
  height: 50px;
</string>
    </property>
    <property name="text">
     <string>Mood Prediction
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelimg">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(img/Standar/user.jpg);

border: 4px solid #4A4A6F;
border-radius: 20px;
padding: 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>310</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(110, 20, 20);</string>
    </property>
    <property name="text">
     <string>Error !</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
